packages/core/RELEASES.md

```diff
@@ -1,5 +1,8 @@
 # @actions/core Releases
 
+### 1.7.0
```

Comment:
might want to rebase to smooth out this diff

Reply:
I tried that and it made this change go away, but for some reason that made the files in `http-client/src` all show up as "added" instead of "modified" (????).  Since those are the files that really matter here, I switched it back to the pre-rebase commit.